numpy/ma/extras.py

```diff
@@ -1524,20 +1545,20 @@ def flatnotmasked_edges(a):
     Examples
     --------
     >>> a = np.ma.arange(10)
-    >>> flatnotmasked_edges(a)
-    [0,-1]
+    >>> np.ma.flatnotmasked_edges(a)
+    array([0, 9])
 
     >>> mask = (a < 3) | (a > 8) | (a == 5)
     >>> a[mask] = np.ma.masked
     >>> np.array(a[~a.mask])
     array([3, 4, 6, 7, 8])
 
-    >>> flatnotmasked_edges(a)
+    >>> np.ma.flatnotmasked_edges(a)
     array([3, 8])
 
     >>> a[:] = np.ma.masked
-    >>> print(flatnotmasked_edges(ma))
-    None
+    >>> np.ma.flatnotmasked_edges(a)
+    >>> # would print None
```

Comment:
Why not leave the `print`?

Reply:
True -- that works now. `print` sometimes causes issues because it will generate an expected value of `random` when the argument is an array